src/localisation/en/en_main.json

```diff
@@ -110,7 +111,7 @@
     "gdriveAllStrypeFiles": "All Strype files",
     "gdriveTab": "My Drive",
     "editorErrors": "Errors",
-    "resyncToGDAtStartup": "Last time you were editing this project, you were saving to Google Drive.\nDo you want to reconnect to Google Drive in order to synchronise your changes?\nIf you do not, you will not have the latest version from Google Drive and your changes will not be saved back there."
+    "resyncToGDAtStartup": "<b>Reconnect and <u>save</u> to Google Drive?</b>\n\nLast time you were editing, you were saving to Google Drive.\nIf you want to <u>load</u> changes from Google Drive, click 'Do not connect' and use 'Open' from the menu."
```

Comment:
I think maybe add the word instead in the second sentence just to help make clear it's an alternative. ("If you want to instead load changes...")